requirements.py

```diff
@@ -0,0 +1,4 @@
+zarr
+ujson
+kerchunk
+netCDF4>=1.6.0
```

Comment:
what's this script for? Also, can we not pin netCDF4 > 1.6.0 - 1.6.1 is problematic, throws random SegFaults

Reply:
https://pip.pypa.io/en/stable/user_guide/#requirements-files. I chose 1.6.0 as the use of the `compression` keyword in `netCDF4.Dataset.createVariable` was only introduced at that version. That said, it's nice (I think) rather than necessary, so it's up to you. GitHub uses it to generate dependency lists (e.g. https://github.com/NCAS-CMS/cf-python/network/dependencies)

Reply:
agh right I thought that was for that. Note we don't use/need this - we have the `environment.yml` file for the conda env and deps are being checked against the deps list in `setup.py` - we really don't want anything from PyPi, but the setup.py contains the deps just so `pip install` gets its satisfaction done. Would you be OK to remove it pls. On the same note we might need a project.toml file, but we're not just there yet

Reply:
clash of titans btw - you are much more PyPi-leaning than me, conda-forge-leaning :grin:

Reply:
Ha! PyPi is a small minnow compared to the bohemoth of conda! Can't ignore the pip folks, though ...

Reply:
ha yeah, still - let's not have it - it adds an extra layer of confusion towards solving the env and such. pip works very well with what we have now :+1:

Reply:
OK - gone.

Reply:
(it was mis-named .py instead of .txt, anyway :))

Reply:
shweet, thank much :beer: A reminder for me to add a `pyproject.toml` file though